### ISSUE TITLE
false positive for VagueImportViolation #WPS347

### ISSUE THREAD

**Issue by vikigenius** [wemake-services/wemake-python-styleguide#1309]
Let us say I have the following structure

```
root_module
  pipeline
    __init__.py
    read.py
    write.py
```

Let us say in `write.py` I try to import `read.py` like this

`from root_module.pipeline import read as read_pipeline  # VagueImportViolation`

Shouldn't it be okay since I am using an alias

**Comment by sobolevn** [comment]
Which version are you using? It is seemed to be fixed in `0.14`

```
» echo 'from root_module.pipeline import read as read_pipeline # VagueImportViolation' > ex.py

» flake8 ex.py
WARNING:flake8.checker:The multiprocessing module is not available. Ignoring --jobs arguments.

ex.py

  1:1      F401  'root_module.pipeline.read as read_pipeline' imported but unused
  from root_module.pipeline import read as read_pipeline # VagueImportViolation
  ^

  1:1      I001  isort found an import in the wrong position
  from root_module.pipeline import read as read_pipeline # VagueImportViolation
  ^

  1:55     E261  at least two spaces before inline comment
  from root_module.pipeline import read as read_pipeline # VagueImportViolation
                                                        ^

Full list of violations and explanations:
https://wemake-python-stylegui.de/en/0.14.0/pages/usage/violations/

```

**Comment by vikigenius** [comment]
@sobolevn You are right. Update to 14.0 fixed this. Closing.
